console: document useSecureCookies (#1673)
redpanda-data/documentation: docs/manage/security/console/plain.mdx

        password: "some-secret-password"
```

:::note

> Disable the `useSecureCookies` option as shown in the above code snippet if Console is exposed via HTTP instead of HTTPS.

Reviewer: ```suggestion
Disable the `useSecureCookies` option shown in the code snippet if Console is exposed using HTTP instead of HTTPS.
```
